app/assets/stylesheets/contrib/RonRonnement-Classic.scss

```diff
@@ -88,15 +89,11 @@ body {
   max-width: 150ex;
   margin: 0 auto;
   border: solid $C_BORDER_FONCE;
-  border-width: 0 1px;
-  background: linear-gradient(
-      90deg,
-      $C_CLAIR,
-      $C_CLAIR $PX_BRANDING_LARG,
-      $C_CONTAINER $PX_BRANDING_LARG,
-      $C_CONTAINER
-    )
-    #fff;
+  border-width: 0 $PX_BODY_BORDER_WIDTH_LEFT_RIGHT;
+  background-repeat: no-repeat;
+  background-size: $PX_BRANDING_LARG + $PX_BODY_BORDER_WIDTH_LEFT_RIGHT, auto;
+  background-position: 0, $PX_BRANDING_LARG + $PX_BODY_BORDER_WIDTH_LEFT_RIGHT;
+  background-image: linear-gradient($C_CLAIR, $C_CLAIR), linear-gradient($C_CONTAINER, $C_CONTAINER);
```

Comment:
Well if you use two gradients why not just use two plain colours?

Reply:
`background-image` CSS rule requires an `image`, so we can't use directly plain colours. The hack is to use `linear-gradiant` to create an image with just the plain color.

Due to the usage of `background-image`, I had to disable the `background-repeat`, otherwise only one of the colour is rendered everywhere.